pkg/controller/job/job_controller.go

```diff
@@ -781,11 +782,15 @@ func (jm *Controller) syncJob(ctx context.Context, key string) (rErr error) {
 	if err != nil {
 		return err
 	}
-
+	var terminating *int32
+	if shouldCountTerminating(&job) {
```

Comment:
can you call Enabled directly here?

Reply:
Sorry, I don't really understand what you are asking?  

do you want me to use a feature toggle here?